dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/report/TrackerErrorReport.java

```diff
@@ -142,12 +138,133 @@ public TrackerErrorReport setValue( Object value )
     @Override
     public String toString()
     {
-        return MoreObjects.toStringHelper( this )
-            .add( "message", getMessage() )
-            .add( "errorCode", message.getErrorCode() )
-            .add( "mainKlass", mainKlass )
-            .add( "errorKlass", errorKlass )
-            .add( "value", value )
-            .toString();
+        return "TrackerErrorReport{" +
+            "message=" + message.getMessage() +
+            ", mainId='" + mainId + '\'' +
+            ", mainKlass=" + mainKlass +
+            ", errorKlass=" + errorKlass +
+            ", errorProperties=" + Arrays.toString( errorProperties ) +
+            ", value=" + value +
+            ", lineNumber=" + lineNumber +
+            '}';
+    }
+
+    public static class Builder
```

Comment:
Can you look into using lombok @Builder for this? https://github.com/dhis2/wow-backend/blob/master/guides/lombok.md